application-admintools-ui/src/main/resources/AdminTools/HelpLinks.xml

```diff
@@ -39,75 +41,52 @@
 {{/box}}
 
 {{velocity}}
+= $escapetool.xml($services.localization.render('adminTools.helppage.performance.title')) =
```

Comment:
I don't see the HTML/XML context. If the wiki syntax were interpreted as HTML then there wouldn't be any need for an HTML macro.

```suggestion
= {{translation key="adminTools.helppage.performance.title" /}} =
```